### DISCUSSION TITLE
Compatability with Fluid Pipes and Item Cables etc.

### DISCUSSION THREAD

**Discussion by zeven1707** [terrarium-earth/Ad-Astra#541] [Ideas]
Currently only a few select blocks/machines can interact with fluid pipes (Create Fluid Pipes, for instance) or Item Transportation Blocks (Simple Storage Network, as an example).
The Fuel Refinery can accept fluid input, however not output. And from testing neither the Oxygen Loader or Fuel Refinery will work with item transportation blocks, like the funnels from Create or Cables from Simple Storage Network.

I thought this was standard, would be really, really nice to have.

Also, cool mod! Reminds me of the good old times with GalactiCraft :)

**Comment by Danekjovax** [comment]
I am also experiencing issues with pushing items into the Oxygen Loader.  No matter the face I push in from with an empty oxygen tank, the item is always inserted into the Water input slot (Upper Left).  Is there some tool to configure this, or does this need a fix?  Please advise.
![image](https://github.com/terrarium-earth/Ad-Astra/assets/10191173/8ab413cd-d5fc-414d-951c-9747fded91c5)
